Sources/Imperial/Services/DeviantArt/DeviantArt.swift

```diff
@@ -6,6 +6,7 @@ public class DeviantArt: FederatedService {
 
     @discardableResult
     public required init(
+        grouped: [PathComponent],
```

Comment:
Add parameter `grouped: [PathComponent]`
It gets passed down in to where the `callback` url is parsed into path components and registered.
Any overlap between callback url path components and `grouped` parameter is removed before registration.
So the grouped route matches the callback url.